docs/howto/handle_errors.md

```diff
@@ -1,6 +1,6 @@
 ---
-id: advanced_error_model
-title:  "Error Model"
+id: howto_handle_errors
+title:  "How to: Handle errors"
```

Comment:
Ah, you don't need to repeat `How to: ` here since the title of the section will be `How to`

Reply:
(same for the others)

Reply:
Argh, I thought that this will only show in page title:
![image](https://user-images.githubusercontent.com/1460799/65966028-dcdce180-e45f-11e9-9127-58e9c812c220.png)

and I didn't understand it will also appear on side bar. Will fix it.